### PR TITLE
Translations experiments

### PR DESCRIPTION
**Pull request by m4tewoosh**
### Description
Whole translations unify, fixes and new option

https://cleeng.atlassian.net/browse/MSSDK-1345
https://cleeng.atlassian.net/browse/MSSDK-1384
https://cleeng.atlassian.net/browse/MSSDK-1300

### Updates
Removed `labeling()` function
Used `useTranslation()` where possible (in functional components)
Added missing translations both in translations.json and in components
Added `Config.setLanguage()` method to change lang without reloading the page
Changed `i18n` config
Added `i18next-http-backend` npm package

### CODE REVIEW

File: src/components/Payment/LegalCopy/LegalCopy.js [hunk 1]
```diff
@@ -18,7 +19,7 @@ const LegalCopy = () => {
   return (
     <LegalNoteWrapperStyled>
       <LegalTextStyled marginBottom="24px">
-        <Trans i18nKey="legal-notes-acknowledge">
+        <Trans i18nKey="legal-notes-acknowledge" t={t}>
```
Comment (iwonakulacz) [inline src/components/Payment/LegalCopy/LegalCopy.js:22]: do we need this 't={t}'?

File: src/components/MyAccountUserInfo/MyAccountUserInfo.js [hunk 2]
```diff
@@ -1,10 +1,7 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import SkeletonWrapper from 'components/SkeletonWrapper';
-
 import { useSelector } from 'react-redux';
-import { withTranslation } from 'react-i18next';
-import labeling from 'containers/labeling';
+import { useTranslation } from 'react-i18next';
+import SkeletonWrapper from 'components/SkeletonWrapper';
```
Comment (pawelacio) [inline src/components/MyAccountUserInfo/MyAccountUserInfo.js:4]: Why this component was added? (SkeletonWrapper)

File: src/i18NextInit.js [hunk 3]
```diff
@@ -15,12 +17,16 @@ i18n
     },
     nsSeparator: false, // namespace separator
     keySeparator: false, // key separator
-    defaultNs: 'translation',
+    ns: ['translations'],
+    defaultNS: 'translations',
     fallbackNS: 'translation',
```
Comment (pawelacio) [inline src/i18NextInit.js:22]: Do we want that `fallbackNS` to be `'translation'` if namespaces changed to `'translations'`?

File: src/util/appConfigHelper.js [hunk 4]
```diff
@@ -162,6 +163,25 @@ export const setHidePayPal = () => {
...
+
+  if (!i18n.hasResourceBundle(language, 'translation')) {
+    const data = await fetch(
+      `${BASE_URL}/cleeng-translations/${language}/translations.json`
+    )
+      .then(response => {
+        return response.json();
+      })
+      .catch(() => {});
+    i18n.addResourceBundle(language, 'translation', data, true, true);
```
Comment (pawelacio) [inline src/util/appConfigHelper.js:177]: If we changed the namespace to `'translations'` is that okey to have here `'translation'` as a namespace in the `hasResourceBundle` and `addResourceBundle`?

File: src/components/MyAccountUserInfo/MyAccountUserInfo.js (hunk 2 above)
Comment (m4tewoosh) [reply src/components/MyAccountUserInfo/MyAccountUserInfo.js:4]: It was there before but import order changed

File: src/components/Payment/LegalCopy/LegalCopy.js (hunk 1 above)
Comment (m4tewoosh) [reply src/components/Payment/LegalCopy/LegalCopy.js]: Now I see it is needed 😅 will revert it back

**Review by pawelacio — DISMISSED** [review]

**Review by pawelacio — APPROVED** [review]

**Pull request merged**